bin/run-jetpack-command.sh

```diff
@@ -32,16 +32,14 @@ pnpm_version=$(npx semver -c "$listed_pnpm_version")
 # More information in: https://github.com/wordpress-mobile/gutenberg-mobile/issues/5688
 sed -i.bak 's/^engine-strict = true/engine-strict = false/' .npmrc
 
-# Install dependecies of Jetpack plugin
-pushd projects/plugins/jetpack
```

Comment:
With the future incorporation of the VideoPress package, seems more practical to install the dependencies from the root level.